services/movie.py

```diff
@@ -28,13 +33,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
The type hint for `genres_ids` and `actors_ids` should be `Optional[list[int]]` to indicate that these parameters can be `None`. This will improve type consistency and clarity.